packages/backend/src/server.ts

```diff
@@ -81,17 +79,27 @@ export function startServer(
     )
 
     apiRouter.post(
-      '/submissions',
-      disableInProduction,
+      '/submissions/:invitationId',
       asyncHandler(async (req, res) => {
-        const payload = req.body
+        const payload = { ...req.body, invitationId: req.params.invitationId }
         logger.info(`Processing submission:`, payload)
         const submission =
           await context.submissionService.addSubmission(payload)
         res.status(201).json(submission)
       }),
     )
 
+    apiRouter.post(
+      '/invite',
```

Comment:
Although not all endpoints is possible to name and use according to REST principles, this one should be possible. It means we are creating `invitation` resource, therefore it should be just `/invitation`, the fact it's creation is defined by the method `POST`.